source/Octopus.Client/Octopus.Client.MacOs.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Octopus.Client</id>
+        <version>$version$</version>
+        <authors>Octopus Deploy</authors>
+        <owners>Octopus Deploy</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <license type="expression">Apache-2.0</license>
+        <projectUrl>https://github.com/OctopusDeploy/OctopusClients</projectUrl>
+        <icon>images\icon.png</icon>
+        <description>Octopus Deploy is an automated release management tool for modern developers and DevOps teams.
+            This package contains the ILmerged client library for the HTTP API in Octopus.</description>
```

Comment:
Description is copied from the release nuspec which is probably wrong. This one doesn't contain an ILMerged client library for example.

Suggestion: Put a developer-specific description in here e.g. "This package contains an unsigned, non-ILMerged, locally built client library for the HTTP API in Octopus"

Reply:
Fixed, good catch!